kernelci/rootfs.py

```diff
@@ -19,8 +19,14 @@
 import os
 
 
-def _build_debos(name, config, data_path):
-    for arch_type in config.arch_list:
+def _build_debos(name, config, data_path, arch_type):
+    arch_list = []
+    if arch_type is None:
+        arch_list = config.arch_list
+    else:
+        arch_list.append(arch_type)
+
+    for arch in arch_list:
```

Comment:
The `_build_debos()` function should be building only one rootfs, so it should just take the `arch` parameter and pass it to `debos`.